gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -51,6 +54,7 @@ export SETUP_HOST="true"
 export SETUP_VIRSH_NET="true"
 export VM_IMAGE_CREATE="true"
 export DEPLOY_OSA="true"
+export OSA_BRANCH="${OSA_RELEASE}"
```

Comment:
Should this be `OSA_RELEASE_BRANCH`?  That's what it seems to be elsewhere in this file.

Reply:
Take a look at the commit messages and it should make sense. `OSA_RELEASE` and `OSA_RELEASE_BRANCH` are two separate things.